common/scripted_effects/cwp_political_setup.txt

```diff
@@ -488,7 +487,7 @@ setup_starting_ideologies = {
 				owner = {
 					or = {
 						has_law = law_type:law_decentralization
-						has_law = law_type:law_informal_schooling
```

Comment:
I remember having to make school law the breakpoint here because it was one of the only differentiating factors between materialistic and traditionalist.